.github/container/Dockerfile.base

```diff
@@ -12,11 +12,15 @@ ARG GIT_USER_NAME
 ARG SRC_MANIFEST_FILE
 ARG DEST_MANIFEST_DIR
 ARG CLANG_VERSION
+ARG NVIDIA_REQUIRE_CUDA
 
 ###############################################################################
 ## Install Python and essential tools
 ###############################################################################
 
+# Temporary WAR to run container on arm64 with CUDA-12.3
+ENV NVIDIA_REQUIRE_CUDA=${NVIDIA_REQUIRE_CUDA}
```

Comment:
Maybe worth adding a `RUN` command here that asserts `NVIDIA_REQUIRE_CUDA` is not empty? I think that otherwise, building this Dockerfile with default arguments would result in a container with an empty `NVIDIA_REQUIRE_CUDA`?

Reply:
Reasonable. Added assert